### PR TITLE
Docker description

### PR DESCRIPTION
**Pull request by thinkh**
@mccalluc Please check if we missed something.

### CODE REVIEW

**Comment by mccalluc** [comment]
Not working for me right now:
```
$ sudo npm install -g yo github:phovea/generator-phovea
Everything looks all right!
/usr/local/lib
├── generator-phovea@0.0.1  (git://github.com/phovea/generator-phovea.git#c0d4ee0639d8544f8d05ca9c732b39cc90591237)
└── yo@1.8.5 
```
```
$ yo phovea_workspace
Error phovea_workspace 

You don’t seem to have a generator with the name “phovea_workspace” installed.
But help is on the way:

You can see available generators via npm search yeoman-generator or via http://yeoman.io/generators/. 
Install them with npm install generator-phovea_workspace.

To see all your installed generators run yo without any arguments. Adding the --help option will also show subgenerators. 

If yo cannot find the generator, run yo doctor to troubleshoot your system.
```

Sorry if I'm missing something obvious?

**Comment by thinkh** [comment]
Try `yo phovea:workspace` (with colon).

**Comment by mccalluc** [comment]
d'oh. Do we want to add a pointer to https://docs.docker.com/compose/install/?

**Comment by thinkh** [comment]
Yes, the link would be a good background information. Add it please.

**Comment by mccalluc** [comment]
```
workspace$ docker-compose up
ERROR: Top level object in './docker-compose.yml' needs to be an object not '<type 'NoneType'>'.
workspace$ cat docker-compose.yml 
workspace$ 
```

After following these steps the file exists but is empty.

**Comment by sgratzl** [comment]
Do you have any repository checked out (like the phovea_server) that contributes to the docker-compose.yml?

**Comment by mccalluc** [comment]
I overlooked the `# initialize/clone/resolve plugins`... Now going back and trying that...

**Comment by mccalluc** [comment]
@thinkh: Do these latest edits make sense? Ideally, if there are choices to be made a comment would be sufficient ... but if someone is skimming (like I was) they may just go from command to command. Also: is the section I added about how to add a plugin to an existing workspace accurate?

**Comment by thinkh** [comment]
Yes, it's good to add it explicitly. Otherwise you can just reference to the readme of phovea_server, which has already the installation instructions.

As for the section "If you realize other Phovea plugins are necessary, halt Docker by hitting ctrl-C, and then:", I would rephrase it to "Adding further dependencies" or something similar.

File: contributors/cheatsheet.md [hunk 1]
```diff
@@ -43,40 +43,53 @@ yo phovea:update
...
+and [Docker Compose](https://docs.docker.com/compose/install/) installed.
 
 ```
 mkdir workspace
 cd workspace
-# initialize/clone/resolve plugins
+
+git clone https://github.com/phovea/phovea_server.git
+# AND/OR clone any other repos that you will need. 
+yo phovea:resolve
```
Comment (mccalluc) [inline contributors/cheatsheet.md:62]: @thinkh: Does it make sense to reference `phovea:resolve` at this point in the workflow?

File: contributors/cheatsheet.md [hunk 2]
```diff
@@ -43,40 +43,53 @@ yo phovea:update
...
 ```
 
-### Launch a Phovea application/service
+To add other Phovea plugins, halt Docker by hitting ctrl-C, and then:
 
 ```
-npm run start:name_of_application
+# For example, if you want to support interactions with a SQL database:
+git clone git@github.com:phovea/phovea_data_sql.git
+yo phovea:resolve
```
Comment (mccalluc) [inline contributors/cheatsheet.md:74]: @thinkh: ditto
